docs/simulations.rst

```diff
@@ -26,19 +26,27 @@ of this file will determine the type of file output (e.g. filebase.fits for
 a FITS file).
 The number of observations to simulate is given by the `--nsim` parameter.
 The filter to use for the completeness function is given by the
-`--compl_filter` parameter.
+`--compl_filter` parameter.  By default, the SEDs are randomly chosen, and
+weighted by their grid+prior weights; this can be changed with the
+`--weight_to_use` parameter.
 
 .. code-block:: console
 
    $ python simulate_obs.py physicsgrid obsgrid outfile \
-                --nsim 200 --compl_filter f475w
+                --nsim 200 --compl_filter F475W
 
 The output file gives the simulated data in the observed data columns
 identified in the physicsgrid file along with all the model parameters
 from the physicsgrid file.  The names of the observed columns are
 `band_rate` and the units are normalized Vega fluxes (to match how
 the observed data are given).
 
+When creating simulated observations, using the standard IMF mass prior will
+skew your catalog to lower-mass stars.  If you wish to sample the full range of
```

Comment:
Is the right word here "full"?  All the IMFs will sample the full range.  Maybe saying that if you want to have an equal number of stars in the simulation at all massed, then use the flat IMF is more accurate.

Reply:
I used "full" because I wasn't getting any stars above ~3 Msun.  But you're right, a flat IMF would be a better way to phrase it.